config/application.rb

```diff
@@ -26,6 +26,11 @@ class Application < Rails::Application
 
     config.active_record.belongs_to_required_by_default = false
 
+    config.active_record.encryption.support_unencrypted_data = true
+
+    config.active_record.encryption.key_derivation_salt = Rails.application.secrets.key_derivation_salt || "test-key-derivation-salt"
```

Comment:
Looks like the variable here would only match production but not development as development is nested in active_record_encryption.